openedx/core/djangoapps/course_apps/rest_api/v1/views.py

```diff
@@ -132,7 +133,13 @@ def get(self, request: Request, course_id: str):
         course_key = CourseKey.from_string(course_id)
         course_apps = CourseAppsPluginManager.get_apps_available_for_course(course_key)
         serializer = CourseAppSerializer(
-            course_apps, many=True, context={"course_key": course_key, "user": request.user}
+            course_apps,
+            many=True,
+            context={
+                "course_key": course_key,
+                "user": request.user,
+                "build_absolute_uri": request.build_absolute_uri,
```

Comment:
I think we can just pass the request, and adapt the serializer to extract both the user and this method from that.

Reply:
Done!